projects/components/src/popover/popover.service.ts

```diff
@@ -25,7 +25,7 @@ export class PopoverService {
     const overlayRef = this.overlay.create({
       positionStrategy: initialPositionStrategy,
       hasBackdrop: options.backdrop,
-      backdropClass: 'cdk-overlay-transparent-backdrop'
+      backdropClass: options.backdrop ? 'modal-overlay-backdrop' : 'cdk-overlay-transparent-backdrop'
```

Comment:
This turns all backdrops dark. If I recall, backdrop class is unused if hasBackdrop is false, so the transparent class would never be used here. We would need to migrate backdrop the config boolean to an enum to the effect of [TRANSPARENT, OPAQUE, NONE] (or use undefined to indicate none)

Reply:
I didn't see any visual difference between None and transparent. Is there one?

Reply:
Visually no, it's a behavior difference - with a transparent backdrop, interaction is blocked with anything outside the popover (and generally we'd close on a click on the backdrop). At least, that was the initial implementation as I recall.

Reply:
oh I see. Makes sense. Will add the new enum state